src/components/Kv/KvCarousel.vue

```diff
@@ -1,24 +1,161 @@
 <template>
-	<div class="kv-carousel">
-		<slot></slot>
+	<div
+		class="kv-carousel"
+		:class="carouselModifierClasses"
+		ref="KvCarousel"
+	>
+		<div class="kv-carousel__slides"
+			@mouseover="pauseOnHover ? paused = true : paused = false"
+			@touchstart="pauseOnHover ? paused = true : paused = false"
+			@mouseleave="pauseOnHover ? paused = false : paused = false"
+			@touchend="pauseOnHover ? paused = false : paused = false"
+			v-touch:swipe.left="previous"
+			v-touch:swipe.right="advance"
+		>
+			<slot :transitionName="transitionName"></slot>
+		</div>
+
+		<div class="kv-carousel__arrows">
+			<button
+				class="kv-carousel__arrows-btn kv-carousel__arrows-btn--left"
+				@click="previous"
+			>
+				<kv-icon
+					class="kv-carousel__arrows-icon"
+					name="small-chevron"
+					:from-sprite="true"
+					title="Show previous slide"
+				/>
+			</button>
+			<button
+				class="kv-carousel__arrows-btn kv-carousel__arrows-btn--right"
+				@click="advance"
+			>
+				<kv-icon
```

Comment:
We might need to adjust these icons. The small chevron is a little thin and could possibly be bolder? There is also the slightest CSS bug: When the right arrow is clicked, the chevron shifts in position slightly then shifts back as the slide transitions.

Reply:
Yeah I'm planning on making a fatter chevron for the KvIcon sprite sheet today. I noticed that right arrow click shift too! so strange 😕